easybuild/easyconfigs/p/PheWAS/PheWAS-0.12-foss-2016b-R-3.3.3.eb

```diff
@@ -0,0 +1,108 @@
+
+easyblock = 'Bundle'
+
+name = 'PheWAS'
+version = '0.12'
+versionsuffix = '-R-%(rver)s'
+
+homepage = 'https://www.vumc.org/cpm/center-precision-medicine-blog/phewas-r-package'
+description = """Provides an accessible R interface to the phenome wide association study."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+
+dependencies = [
+    ('R', '3.3.3', '-X11-20160819'),
+]
+
+exts_defaultclass = 'RPackage'
+exts_list = [
+    ('meta', '4.9-0', {
+        'source_tmpl': '%(name)s_%(version)s.tar.gz',
+        'source_urls': ['http://cran.r-project.org/src/contrib/'],
```

Comment:
@vanzod You'll also need to include `https://cran.r-project.org/src/contrib/Archive/meta/` here (same goes for other extensions)...

Or, you can specify default extension options with the general `exts_default_options` easyconfig parameter, and only override where needed:

```python
exts_default_options = {
    'source_urls': [
        'http://cran.r-project.org/src/contrib/Archive/%(name)s',  # package archive
        'http://cran.r-project.org/src/contrib/',  # current version of packages
        'http://cran.freestatistics.org/src/contrib',  # mirror alternative for current packages
    ],
    'source_tmpl': '%(name)s_%(version)s.tar.gz',
}
```